tests/console/yast2_i.pm

```diff
@@ -18,13 +18,34 @@ use testapi;
 use utils;
 use version_utils qw(is_tumbleweed is_jeos is_sle);
 
+sub set_action {
+    my $action = shift;
+    return unless defined($action);
+    assert_script_run qq{sed -i 's/PKGMGR_ACTION_AT_EXIT=.*/PKGMGR_ACTION_AT_EXIT="$action"/' /etc/sysconfig/yast2};
```

Comment:
recently I wrote a function to simplify `sed'ing` files , just recommend you to take a look on the code it is in utils.pm
with my function your code will look like this :
```suggestion
    file_content_replace('/etc/sysconfig/yast2', 'PKGMGR_ACTION_AT_EXIT=.*' =>'PKGMGR_ACTION_AT_EXIT="$action"');
```

Reply:
This way ``$action`` would be interpreted as a bash variable, which is not my intention. 
But I like your wrapper! Thumbs up!